docs/bridge/spec.md

```diff
@@ -0,0 +1,991 @@
+Both are independent ledgers. They have their own validators and can have their
+own set of custom transactions. Importantly, there is a way to move assets from
+the locking-chain to the issuing-chain and there is a way to return those assets
+from the issuing-chain to the locking-chain. This key operation is called a
+cross-chain transfer. A cross-chain transfer is not a single transaction. It
+happens on two chains, requires multiple transactions and involves an additional
+server type called a "witness". This key operation is described in a section
+below.
+
+This bridge does not exchange assets between two ledgers - instead it locks
+assets on one ledger (the locking-chain) and represents those assets with
+wrapped assets on another chain (the issuing-chain). A good model to keep in
+mind is a box with in infinite supply of wrapped assets. Putting an asset from
+the locking chain into the box will release a wrapped asset onto the issuing
+chain. Putting a wrapped asset from the issuing chain back into the box will
+release one of the existing locking-chain assets back onto the locking chain.
+There is no other way to get assets into or out of the box. Note that there is
+no way for the box to "run out of" wrapped assets - it has an infinite supply.
+
+After the mechanics of a cross-chain transfer are understood, an overview of
+some supporting transactions are described. This includes transactions to create
+bridges, add attestations from the witness servers, create accounts across
+chains, and change bridge parameters.
+
+The next sections describe the witness server, how to set-up a bridge, error
+handling, and trust assumptions.
+
+Finally, a low-level description of the new ledger objects and transactions are
+presented as a reference.
+
+## Nomenclature
+
+Cross-chain transfer: A transaction that moves assets from the locking-chain to
+the issuing-chain, or returns those assets from the issuing-chain back to the
+locking-chain.
+
+Claim ID: A ledger object used to prove ownership of the funds moved in a
+cross-chain transfer.
+
+Account-create ordering number: A counter on the door accounts used to order the
+account create transactions.
+
+Door accounts: The account on the locking-chain that is used to put assets into
+trust, or the account on the issuing-chain used to issue wrapped assets. The name
+comes from the idea that a door is used to move from one room to another and a
+door account is used to move assets from one chain to another.
+
+Locking-chain: Where the assets originate and are put into trust.
+
+Issuing-chain: Where the assets from the locking-chain are wrapped.
+
+Witness server: A server that listens for transactions on one or both of the
+chains and signs attestations used to prove that certain events happened on a
+chain.
+
+## List of new transactions
+
+- XChainCreateBridge
+- XChainModifyBridge
+- XChainCreateClaimID
+- XChainCommit
+- XChainClaim
+- XChainCreateAccountCommit
```

Comment:
This was renamed to `XChainAccountCreateCommit`.

Reply:
d10653757f `[fold] Remove obsolete spec`